### ISSUE TITLE
[Feature Engineering] Normalização de Escala (StandardScaler)

### ISSUE THREAD

**Issue by davi-balsamao** [williamalmeidadev/deepfake-detection-research#5]
Ajuste matemático das colunas numéricas para que todas fiquem dentro da mesma faixa de grandeza.

Por que estamos fazendo assim? Se uma coluna tem notas de 0 a 10 e outra de 0 a 100.000, o algoritmo assumirá falsamente que a coluna de grandeza maior é mais importante. A normalização corrige essa assimetria.

Decisão Técnica: Aplicaremos a Z-Score Normalization (StandardScaler), que transforma os dados garantindo média igual a 0 e desvio padrão igual a 1. É mandatório antes do PCA

Código de Apoio:
```py
from sklearn.preprocessing import StandardScaler
scaler = StandardScaler()
# Explicação: Nunca fitamos (treinamos) o scaler na variável alvo (Target/Y). Scaler roda apenas nas variáveis independentes (Features/X).
df_features_scaled = scaler.fit_transform(df_features)
```

Critérios de Aceite (Definition of Done):
- O dataframe final deve possuir variáveis numéricas com `mean() ~ 0` e `std() ~ 1`.
- Dataframe validado e consolidado ser exportado como um novo arquivo CSV (`deepfake_dataset_cleaned.csv`) para uso nas fases seguintes.

Checklist:
- [x] Importar StandardScaler do `sklearn.preprocessing`.
- [x] Isolar as colunas numéricas em um dataframe de features (X).
- [x] Aplicar `fit_transform` nas colunas numéricas
- [x] Exportar a base final tratada.
- [x] Agendar validação do dataset com os professores (Meta: 03/03).
